src/Components/Movies/singleMovie.js

```diff
@@ -1,17 +1,29 @@
 import React from 'react'
+import PropTypes from 'prop-types'
 
-
-const SingleMovie = ({filteredMovie, returnHome}) => {
-  return (
-    <div className='single-movie' >
-      <img className='single-poster' src={filteredMovie[0].backdrop_path} ></img>
-      <div className='text-details'>
-        <p>{filteredMovie[0].title}</p>
-        <p>{filteredMovie[0].average_rating}</p>
-        <button onClick={() => returnHome()} >Return</button>
+const SingleMovie = ({filteredMovie, returnHome}) =>  {
+  console.log('filtered', filteredMovie)
+    return (
+      <div className='single-movie' >
+        <img className='single-poster' src={filteredMovie.backdrop_path} ></img>
+        <div className='text-details'>
+          <p>{filteredMovie.title}</p>
+          <p>{filteredMovie.average_rating}</p>
+          <p>{filteredMovie.budget}</p>
+          <p>{filteredMovie.overview}</p>
+          <p>{filteredMovie.runtime}</p>
+          <p>{filteredMovie.release_date}</p>
+          <button onClick={() => returnHome()} >Return</button>
+        </div>
       </div>
-    </div>
-  )
-}
+    )
+  }
+
+  SingleMovie.propTypes = {
+    filteredMovie: PropTypes.object.isRequired
+    
```

Comment:
is it okay to just have 1 proptype to check a whole object we are passing through? This doesn't seem like a whole lot of work. 'Too good to be true'

Reply:
I'm glad you asked this.  Think about what PropTypes are used for?  Is this accomplishing that goal thoroughly?  Check out the docs.  Are their more options to use PropTypes to check for specific data types within an object?  Ultimately you get to decide.